.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "control libraries",
+    "remoteUser": "ros2",
+    "build": {
+        "dockerfile": "../Dockerfile.ci",
+        "context": "..",
+        "target": "development",
+        "args": {
+            "ROS2_VERSION": "humble",
```

Comment:
This arg is not used by the CI dockerfile, the base image is hardcoded. On a side-note, it would probably be more correct to use an `ubuntu` base image rather than `ghcr.io/aica-technology/ros2-ws:humble` anyway.

Reply:
Yeah we could use ubuntu but then we end up installing a bunch of apt packages (cmake, make, complier) that we already have in the workspace?

Reply:
can't we use the existing development-dependencies image then? I just think it's weird to see ROS stuff here

Reply:
I can do that yeah, I just think that the development dependencies image could be removed at some point because its very heavy and apart from potentially here, it's not used anywhere. `ubuntu` is slightly annoying because we'd have to configure a user again.